networking_ccloud/ml2/agent/common/messages.py

```diff
@@ -188,6 +197,16 @@ class SwitchConfigUpdate(pydantic.BaseModel):
     def make_object_from_net_data(self, vxlan_map, net_host_map):
         pass
 
+    def sort(self):
+        # in case we want to compare this or ship it to the user we may benefit from it being sorted
+        # also test cases would fail otherwise
+        if self.vlans:
+            self.vlans.sort()
+        if self.vxlan_maps:
+            self.vxlan_maps.sort()
+        if self.ifaces:
+            self.ifaces.sort()
+
```

Comment:
Shouldn't we sort all members of `SwitchConfigUpdate`? `self.bgp` would need some sorting as well.